app/policies/presentation_policy.rb

```diff
@@ -6,4 +6,12 @@ def create?
   def new?
     create?
   end
+
+  def update?
+    person.present? && (person.admin? || person.presentations.include?(record))
```

Comment:
Alright I've landed on this solution

```
(person&.admin? || person&.presentations&.include?(record)).present?
```

You can't get an empty string or empty array back from and or clause and this avoids the double !